Firmware/LowLevel/src/datatypes.h

```diff
@@ -135,22 +137,62 @@ struct ll_ui_event {
 } __attribute__((packed));
 #pragma pack(pop)
 
-#define LL_HIGH_LEVEL_CONFIG_MAX_COMMS_VERSION 1           // Max. comms packet version supported by this open_mower LL FW
 #define LL_HIGH_LEVEL_CONFIG_BIT_DFPIS5V 1 << 0            // Enable full sound via mower_config env var "OM_DFP_IS_5V"
-#define LL_HIGH_LEVEL_CONFIG_BIT_EMERGENCY_INVERSE 1 << 1  // Sample, for possible future usage, i.e. for SA-Type emergency
+#define LL_HIGH_LEVEL_CONFIG_BIT_BACKGROUND_SOUNDS 1 << 1  // Enable background sounds
 
 typedef char iso639_1[2]; // Two char ISO 639-1 language code
 
+enum class HallMode : uint8_t {
+    OFF = 0,
+    EMERGENCY,  // Triggers emergency with lift & tilt functionality
+    STOP,       // Stop mower
+    PAUSE       // Pause the mower (not yet implemented in ROS)
+};
+
+// FIXME: Decide later which is more comfortable, activeLow = 0 | 1
```

Comment:
When I add a `FIXME` comment, it usually means stuff that I want to address before merging - is there still anything to decide right now?

Reply:
Me too, but in my understanding something which needs to be fixed before merging (if technical possible).

And "yes", a loot. See my top post as well as this PR is in "Draft"

Reply:
I was only referring to decisions about this single line 😆 IIUC, it's just about whether to encode ACTIVE_LOW as value 0 or 1? Shouldn't matter, but 0 makes more sense to me.

Reply:
I also referred about that single line :-)

No wouldn't matter much, but like to decide later because even if ACTIVE_LOW as 0 sound logic, it isn't, because ACTIVE_LOW also means "inverted". Have had the enum at first as "normal" and "inverted" but within the code when working with the GPIOs, "normal" is somehow undefined. That's why I switched to the more technical word "Active Low" and "Active High" because it's more precise and normally clear for a developer.

Later on, within mower_config, I think we shouldn't use active_* because to technical for the normal user. The normal user knows better with "inverted" or "normal" (non-inverted).

And lastly: later when building the emergency logic, it should also look straight forward and not popup questions like "Why does he invert that again when it's already active_high?".
I don't waste thinking about it ATM, will decide later when I'm clear with the code, and changing this enums later is the simplest ;-)